src/coq/Handlers/Serialization.v

```
@@ -141,12 +141,42 @@ Module Type SerializationBase (LP : LLVMParams) (MP : MemoryParams LP).
               | None => extractbytes_to_dvalue M undef_handler ERR_M lift_ue bytes dt
               end
             else extractbytes_to_dvalue M undef_handler ERR_M lift_ue bytes dt
+        | UVALUE_InsertValue t uv elt idxs =>
+            str <- concretize_uvalueM M undef_handler ERR_M lift_ue uv;;
+            elt <- concretize_uvalueM M undef_handler ERR_M lift_ue elt;;
+            let fix loop str idxs : ERR_M dvalue :=
+              match idxs with
+              | [] => ret str (* What to do? Do we print out some error *)
```

This is a good point, let's make this raise an error instead of failing silently and terrifyingly.